_posts/blog/2015-11-08-gtac.md

```diff
@@ -15,13 +15,87 @@ date: 2015-11-08T15:39:55-04:00
 
 ## GTAC 2015
 
-I had the honor to be invited to the GTAC 2015 in Boston. It was my first visit to Boston and I have to admit it is a real beauty. I arrived one day before the GTAC started so I had the chance to walk through this tremendeous city. It is a great feeling to sit in the morning sun on the venerable Harvard campus and watch how the future elite trots to their lectures. 
+I had the honor to be invited to the GTAC 2015 in Boston. It was my first visit to Boston and I have to admit it is a real beauty. I arrived one day before the GTAC started so I had the chance to walk through this tremendeous city. It is a great feeling to sit in the morning sun on the venerable Harvard campus and watch how the future elite trots to their lectures.
+
+<figure class="third">
+	<a href="/images/GTAC_2015_harvard.jpg"><img src="/images/GTAC_2015_harvard.jpg" alt="image"></a>
+	<a href="/images/gtac_2015_charles.jpg"><img src="/images/gtac_2015_charles.jpg" alt="image"></a>
+	<a href="/images/gtac_2015_MIT.jpg"><img src="/images/gtac_2015_MIT.jpg" alt="image"></a>
+	<figcaption>Some impressions of Boston.</figcaption>
+</figure>
+
+I also had to spend some time at Primark, because Air Berlin managed to forget my luggage in Berlin and I needed some clothes... But this is a different story.
```

Comment:
Thanks to Air Berlin, I also had to spend some time at Primark as they forget my luggage in Berlin ... But this is a different story.